third_party/openssl/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ if(ANDROID)
         PATCH_COMMAND patch -p 0 < ${PROJECT_SOURCE_DIR}/dockcross-android.patch
         CONFIGURE_COMMAND ${CMAKE_COMMAND} -E env ANDROID_NDK=$ENV{CROSS_ROOT} <SOURCE_DIR>/Configure --sysroot=$ENV{CROSS_ROOT}/sysroot ${OPENSSL_PLATFORM} --prefix=${CMAKE_INSTALL_PREFIX} no-shared
         BUILD_COMMAND make
+        INSTALL_COMMAND make install_sw
```

Comment:
what's that? :laughing:

Reply:
Oh, that was just a try to prevent it from installing tons of docs every time but it did not work. Let me remove it again.